codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/UnconstrainedCollectionGenerator.kt

```diff
@@ -94,33 +98,5 @@ class UnconstrainedCollectionGenerator(
                 "TryFrom" to RuntimeType.TryFrom,
             )
         }
-
-        modelsModuleWriter.withInlineModule(constraintViolationSymbol.module()) {
-            // The first component of the tuple struct is the index in the collection where the first constraint
```

Comment:
Might be worth moving this comment to the new constraint violation type in `CollectionConstraintViolationGenerator`.

Reply:
Good shout. I've chosen to add this directly as docs for the `Member` variant.
Done in https://github.com/awslabs/smithy-rs/pull/2028/commits/9e8c22b9eb29e401086ab20266b9dc1489fffa60.